packages/reactive_ble_mobile/lib/src/generated/bledata.pbenum.dart

```diff
@@ -0,0 +1,7 @@
+///
```

Comment:
See other comments about generated side effects, for this file and all the other new ones in this directory